docs/StepbyStepManualDeploymentGuideScalarDLOnAzure.md

```diff
@@ -0,0 +1,237 @@
+        * Availability set: Select created availability set from dropdown menu
+        * Availability zone: Select an availability zone 1 or 2 or 3
+    * Select Image _CentOS-based 7.9 - Gen 1_
+    * Select **Size** as _Standard_B1s_ from dropdown menu
+    * Select authentication type as SSH public key(generate an SSH key to access the server) or password(can use username and password to SSH access the server).
+    * Enter **Username** as centos
+    * If you are using **Password** for authentication type.
+        * Enter **Password**
+        * Enter **Confirm password**
+    * If you are using **SSH public key** for authentication type.
+        * Select **Generate new key pair**
+        * Select **Use existing public key** and upload already generated public key in SSH public key box.
+    * Select **Public inbound ports** as _Allow selected ports_.
+    * **Select inbound ports** as _SSH (22)_ from the dropdown menu.
+* On the Networking page, select virtual network settings
+    * Select your **virtual network** from drop-down menu
+    * Select subnet **public**
+    * Click **Review+Create**
+    * Click **Create**
+
+## Create Kubernetes Cluster
+
+You can use the following steps to configure Kubernetes cluster based on the [requirements](./ManualDeploymentGuideScalarDLOnAzure.md#step-3-configure-aks)
+
+* Select [Kubernetes services](https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.ContainerService%2FmanagedClusters) from Azure portal
+* Click **Create**
+* Select **Create a Kubernetes cluster**
+* On the Basics page, configure the following options
+    * Select your **Subscription**
+    * Select your **Resource group**
+    * Enter **Kubernetes Cluster name**
+    * Select **Region**
+    * Select **Availability zones** from drop down if you want to create in multiple availability zones
+    * Select **Kubernetes version** _1.19.11_ from dropdown
+    * Select **Node size** as _Standard_D2s_v3_
+    * Select **Scale method** as _Autoscale_
+    * Select Node count range as min: 3 and max: 6
+    * Click **Next: Node pools**
+* On the Node Pools page, configure the following options
+    * Click **Add node pool**
+    * On Add a node pool page, configure the following options
+        * Enter **Node pool name** as _scalardlpool_
+        * Select **Mode** as _User_
+        * Select **OS type** as _Linux_
+        * Select **Availability zones** from drop-down
+        * Select **Node size** as _D2s_v3_
+        * Select **Node count** range as min: 3 and max: 6
+        * Enter **Max pods per node** as 10
+        * Click **Add**
+        * Click **Next: Authentication**
+* On the Authentication page, configure the following options
+    * Select _Service principal_ as **Authentication method**
+    * Enter **Service principal client ID** (Use appId of service principal)
+    * Enter **Service principal client secret** (Use password of service principal)
```

Comment:
I think we should customize the values of `Kubernetes service address range` and `Kubernetes DNS service IP address`